packages/hub/package.json

```
@@ -21,9 +21,8 @@
 	},
 	"source": "index.ts",
 	"scripts": {
-    "prebuild": "pnpm --filter 'docs-internal' prepublish-hub",
-		"format": "prettier --write . && eslint --quiet --fix --ext .cjs,.ts .",
-		"prepublishOnly": "npm run build",
+    "format": "prettier --write . && eslint --quiet --fix --ext .cjs,.ts .",
+		"prepublishOnly": "npm build && pnpm update-docs",
```

```suggestion
		"prepublishOnly": "npm build",
```

I don't think this should be here - I may be mistaken but as I understand, changesets:

1. opens a PR
2. keeps it up to date
3. after merging the PR, it publishes

We want to generate the docs in step 2., before the PR is merged (so the doc changes can be included)

---

Yes! This for sure shouldn't be there. Thanks for catching it